src/ipv4.c

```diff
@@ -122,6 +122,25 @@ static int ipv4_get_route(struct rtentry *route)
 	uint32_t rtdest, rtmask, rtgtw;
 	int rtfound = 0;
 
+	/* initialize the buffer with zeroes, aiming to address the
+	 * coverity issue "tainted_skalar passed to a tainted sink"
```

Comment:
A minor typo here: skalar → scalar

I suggest either of:
"tainted scalar passed to a tainted sink"
"TAINTED_SCALAR passed to a tainted sink"

Reply:
I have picked the second option